vocabularies/survey-type.ttl

```diff
@@ -172,128 +83,23 @@ svytp:physical-geology a skos:Concept ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "em" ;
-    skos:prefLabel "Electromagnetic"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:geochemistry a skos:Concept ;
-    skos:definition "A survey where the chemical properties of rocks, soil, or other geological material is assessed to determine mineralogical and/or elemental composition."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "gc" ;
-    skos:prefLabel "Geochemistry"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:photographic a skos:Concept ;
-    skos:definition "A survey using standard photographic equipment for image capture without intention for spectral analysis. Including land-based geological photography."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "ph" ;
-    skos:prefLabel "Photographic"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:radiometric a skos:Concept ;
-    skos:altLabel "Gamma Logs"@en,
-        "Gamma-Ray Spectrography"@en ;
-    skos:definition "A survey where the emissions of natural gamma-rays are recorded to determine the quantity of radioelements (Potassium, Uranium, Thorium)."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "rm" ;
-    skos:prefLabel "Radiometric"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:spectral a skos:Concept ;
-    skos:altLabel "hyperspectral"@en ;
-    skos:definition "A survey capturing image information and intensity-per-wavelength-band to be used for spectral analysis e.g. ASTER, MODIS."@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "sp" ;
-    skos:prefLabel "Spectral"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
 svytp:spatial a skos:Concept ;
     skos:definition "A survey measuring the spatial characteristics of an object or region."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
     skos:notation "st" ;
     skos:prefLabel "Spatial"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:gravity a skos:Concept ;
-    skos:definition "A survey where the variation of the Earth's gravity field in measured to determine the distribution of density in the region"@en ;
-    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
-    skos:notation "gv" ;
-    skos:prefLabel "Gravity"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:magnetic a skos:Concept ;
-    skos:altLabel "aeromagnetics"@en ;
-    skos:definition "A survey where the variations of the Earth's magnetic field is measured to determine the distribution of magnetic minerals (predominately magnetite)"@en ;
+    
+svytp:spectral a skos:Concept ;
+    skos:altLabel "hyperspectral"@en ;

```

Comment:
add new altlabel: multispectral

Reply:
Spectral needs a notation. 
Also, Idk about the logic behind using altlabels as opposed to narrower terms for multispectral (aster) and hyperspectral (hymap).  Without these as narrower terms. how can I facet my search results to weed out the things I don't want? I see that hyper and multi are here as observation_types. Do we have a ckan profile for observation, or is the obs_type somewhere on the survey profile?

Reply:
I'll add skos:notation "sp" ;
I have the altLabels there. But I'm not super comfortable having them as double-ups with the observation types. Really it should just be Survey = spectral, observation = hyperspectral
where you'd facet your search at the observation level. same as how you'd distinguish between a mag and a grav survey that are both just geophysical at the survey level.

https://github.com/geological-survey-of-queensland/gsq-observation-profile/tree/review_observation_profile

Reply:
my 2 cents on this is that like geophysics we don't care at the survey level how many bands they are observing.   Thats all obs type/method stuff.  The alt label was added as people refer to all spectral surveys often as hyperspectral regardless of the type of survey

Reply:
@geoderekh Yep I've already let David know we'll need a 'Spectral' CKAN profile based off the vanilla 'grav' survey profile. Observation Method/Instrument fields should enable people to refine searches for Multi/Hyper data depending on where they've been included in the Vocabs.